pkg/registry/resource/resourceclaim/strategy.go

```diff
@@ -119,7 +136,10 @@ type resourceclaimStatusStrategy struct {
 	resourceclaimStrategy
```

Comment:
Make this a pointer.

```suggestion
	*resourceclaimStrategy
```